packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hds-react",
-  "version": "0.21.0",
+  "version": "0.21.1",
```

Comment:
This change can be omitted; we will update the version when we make a release

Reply:
fixed